Makefile

```diff
@@ -40,7 +40,7 @@ test-data:
 	mkdir -p ${tempDir}
 	git clone -b ${branchName} --depth 1 --single-branch ${githubRepoLink} ${gitDataDir}
 	cp -r ${gitDataDir}/ufc ${testDataDir}
-	rm ${testDataDir}/ufc/bandit-tests/*.dynamic-typing.json
+	rm -f ${testDataDir}/ufc/bandit-tests/*.dynamic-typing.json || true
```

Comment:
bumped into this failing when I was doing some local testing that involved temporarily deleting most of the test data/cases. We shouldn't bump over this in any real scenario but no need to have this fail at any rate